.github/workflows/pr-linter.yml

```diff
@@ -1,6 +1,6 @@
 name: pull request linter
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
I think we need both `pull_request` and `pull_request_target`. because for example this pull request wasn't check.

Reply:
It was not supposed to be the case, checking.